### DISCUSSION TITLE
How to enter a valid system Date in yaml header in Quarto

### DISCUSSION THREAD

**Discussion by BB1464** [quarto-dev/quarto-cli#6596] [Q&A] [answered]
### Description

Good Morning All,
  I know in Rmarkdown I can enter the current date like this: "`r format(Sys.time(), '%d %B, %Y')`"; but now what I am looking for is that once the quarto document is rendered I want it to show the current date and time that the document is rendered. Thank you all.

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/reference/dates.html>.

**Comment by BB1464** [comment]
Thank you very much @mcanouil this is what I was looking for
